docs/content/users/extend/additional-services.md

```diff
@@ -79,29 +79,29 @@ For example,
 
 Officially-supported add-ons:
```

Comment:
Overall, we want to do better with this, of course. It needs to be automatic, and it's a bit of a mess anyway. And we need a registry web page. Ideas welcome.